src/request/sign-transaction/SignTransactionApi.js

```diff
@@ -66,9 +118,9 @@ class SignTransactionApi extends TopLevelApi {
     }
 }
 
-/** @type {{[layout: string]: string}} */
+/** @enum {KeyguardRequest.SignTransactionRequestLayout} */
 SignTransactionApi.Layouts = {
-    STANDARD: 'standard',
-    CHECKOUT: 'checkout',
-    CASHLINK: 'cashlink',
+    STANDARD: /** @type {'standard'} */ ('standard'),
```

Comment:
Why is the annotation required?

Reply:
It is an enum over type `KeyguardRequest.SignTransactionRequestLayout` which includes only the constants 'standard', 'checkout' and 'cashlink'. Therefore, the strings need to be casted to these constant values.